packages/react/src/components/uiComponents.tsx

```diff
@@ -301,27 +379,64 @@ export const CreateOrganization = withClerk(({ clerk, ...props }: WithClerkProp<
   );
 }, 'CreateOrganization');
 
+const OrganizationSwitcherContext = createContext<MountProps>(
+  // @ts-expect-error We are defining the values below
+  {},
+);
+
 const _OrganizationSwitcher = withClerk(
   ({ clerk, ...props }: WithClerkProp<PropsWithChildren<OrganizationSwitcherPropsWithoutCustomPages>>) => {
     const { customPages, customPagesPortals } = useOrganizationProfileCustomPages(props.children);
     const organizationProfileProps = Object.assign(props.organizationProfileProps || {}, { customPages });
 
+    const passableProps = {
+      mount: clerk.mountOrganizationSwitcher,
+      unmount: clerk.unmountOrganizationSwitcher,
+      updateProps: (clerk as any).__unstable__updateProps,
+      props: { ...props, organizationProfileProps },
+    };
+
+    /**
+     * Prefetch organization list
+     */
+    clerk.__internal_prefetchOrganizationSwitcher();
```

Comment:
Previously the Trigger would prefetch those. Since we don't have the trigger when `asStandalone: true` we need to trigger this manually.

Calling `useOrganizationList()` inside here would not work, as clerk-react and clerk-js have a different swr cache (since it's bundled twice)